### PR TITLE
Minor cleanups / refacorings / documentation improvements

### PR DESCRIPTION
**Pull request by Geod24**
By-product of many, many hours spent in the Ledger.

### CODE REVIEW

File: source/agora/node/FullNode.d [hunk 1]
```diff
@@ -1139,7 +1139,7 @@ public class FullNode : API
 
     /// GET: /preimages_from
     public override PreImageInfo[] getPreimagesFrom (ulong start_height)
-        @safe nothrow
+        @safe
```
Comment (linked0) [inline source/agora/node/FullNode.d]: This line could be deleted by adding `@safe` in the previous line.

**Review by linked0 — COMMENTED** [review]
It seems that the `unittest`s of EnrollmentManager still need the `getEnrollment`. :)

**Comment by Geod24** [comment]
Rebased, should be fixed

File: source/agora/node/FullNode.d (hunk 1 above)
Comment (Geod24) [reply source/agora/node/FullNode.d]: Did it.

**Review by omerfirmak — APPROVED** [review]

**Review by linked0 — APPROVED** [review]
LGTM

**Pull request merged**